Service/NotificationManager.php

```diff
@@ -0,0 +1,170 @@
+<?php
+  /**
+   * Created by groovili
+   * https://github.com/groovili
+   */
+  declare(strict_types = 1);
+
+  namespace Kopay\NotificationBundle\Service;
+
+  use Kopay\NotificationBundle\Entity\NotificationEmail;
+  use Kopay\NotificationBundle\Entity\NotificationMessageInterface;
+  use Kopay\NotificationBundle\Entity\NotificationPush;
+  use Symfony\Component\OptionsResolver\Exception\InvalidArgumentException;
+
+  /**
+   * Class NotificationManager
+   *
+   * @package Kopay\NotificationBundle\Service
+   */
+  class NotificationManager {
+    /**
+     * @var array
+     */
+    private $configuration;
+
+    protected const NOTIFICATION_EMAIL = 0;
```

Comment:
Why 0/1 as constant values? What if SMS will be next notification, should we add magic 3? Please use meaningful string values - they are more verbose.

Reply:
ok